unused/war_over_the_heavenly_horses.xml

```diff
@@ -0,0 +1,8 @@
+In the second century BC, the Han faced a big threat: a nomadic people called Xiognu, who inhabited the lands north of the Han dynasty, were continuously raiding the border regions. The Xiognu were horsemen and because the Han infantry and chariots proved too slow to chase them. For almost a century, the Han were forced to send massive tributes to the Xiognu to keep the peace. However, the Xiognu kept pillaging nonetheless.
+Only Emperor Wu decided to finally put an end to it and declare war on the Xiognu. And to counter the Xiognu horsemen, he planned to raise a big cavalry force himself. There was only one problem: one of the few natural resources that the mighty Han dynasty lacked were powerful horses suitable for cavalry warfare. Local breeds were simply too small and weak. And large parts of the Empire were everything but ideal for horses grazing.
+But the news Zhang Qian brought came in very convenient: Zhang Qian was a Han envoy sent by Emperor Wu himself to explore and find allies in the West. He had, after over a decade, finally returned and although failed to make allies, brought invaluable information about previously unknown lands. He had, among others, also come to Greco-Bactria. Greco-Bactria was, because Alexander the Great had conquered this region and established colonies there not even 200 years prior, inhabited by predecessors of Alexander's soldiers, hence by Greeks. And Zhang Qian reported seeing extremely strong horses there—exactly what the Han needed. The horses he had seen were the mystical breed of so-called Ferghana horses. They were comparably small and had short legs, but they were very fast, powerful, and endurable. They were even said to have sweated blood. In fact, Zhang Qian was so thrilled by the Ferghana horses that he called them “Heavenly Horses”. Emperor Wu immediately sent an envoy, trying to buy some horses. However, the Greeks declined and killed the envoy. It is not entirely sure why, but the Ferghana horses were probably considered national pride and not to be sold. Emperor Wu sent his general Li Guangli with army of 20.000 infantry and 6.000 cavalry to take the horses by force. However, to get to Greco-Bactria, Li Guangli had to traverse the hostile Gobi and Taklamakan deserts. He also got into fights with the local Tocharians. And although he eventually arrived at its target, his army was very decimated and Li Guangli was forced to withdraw back to China. 
```

Comment:
```suggestion
But the news Zhang Qian brought came gave hope. Zhang Qian was a Han envoy sent by Emperor Wu to explore and find allies in the West. He had, after over a decade, finally returned and although failed to make allies, the envoy brought invaluable information about unknown lands, namely Greco-Bactria. Greco-Bactria was, because Alexander the Great had conquered this region and established colonies there not even 200 years prior, inhabited by predecessors of Alexander's soldiers, Greeks. Thrtr Zhang Qian reported seeing extremely strong horses there—exactly what the Han needed. The horses he had seen were the mystical breed of so-called Ferghana horses. They were comparably small and had short legs, but they were very fast and powerful. They were even said to have sweated blood. Zhang Qian was so thrilled by the Ferghana horses that he called them “Heavenly Horses”. 
Emperor Wu immediately sent an envoy, trying to buy some horses. However, the Greeks refused and killed the envoy. It is not entirely sure why, but it was possible that the Ferghana horses were a national pride and not to be sold. Emperor Wu sent his general Li Guangli with an army of 20,000 infantry and 6,000 cavalry to take the horses by force. However, to get to Greco-Bactria, Li Guangli had to traverse the hostile Gobi and Taklamakan deserts. He also got into fights with the local Tocharians. When he eventually arrived, his army was very decimated and Li Guangli was forced to withdraw back to China. 
```

This paragraph is so big that GitHub isn't showing the diffs between changes. You'll have to read it carefully, but most of the changes just make it easier to read.

One thing to note, is that English uses commas (,) to separate thousands, not periods (.)